tasks/pipeline.py

```diff
@@ -160,7 +170,8 @@ def notify_failure(_, notification_type="merge"):
     elif notification_type == "deploy":
         header = ":host-red: :rocket: datadog-agent deploy"
 
-    failed_jobs = get_failed_jobs("DataDog/datadog-agent", os.getenv("CI_PIPELINE_ID"))
+    project_name = "DataDog/datadog-agent"
+    failed_jobs = get_failed_jobs(project_name, os.getenv("CI_PIPELINE_ID"))
 
     # Take care of the global message that goes in the common channel
     message = prepare_global_failure_message(header, failed_jobs)
```

Comment:
Another note for later: should the global failure message also contain the test failure details? In which case we can refactor the global failure message creation.

Reply:
I didn't do this because it wasn't super easy right now, but we can definitely do it if we refactor what you mentioned in your comment above